### PR TITLE
Added a configuration popup to enable performance collection for a storage device

### LINKED ISSUE #404: [Delfin UI]  Performance Monitoring metrics collection configuration page
**Issue/Feature Description:**
Allow the user to configure the parameters for  performance metrics collection

**Why this issue to fixed / feature is needed(give scenarios or use cases):**
The performance monitoring metrics collection has to be configured with different parameters for different types of resources. 

**How to reproduce, in case of a bug:**

**Other Notes /  Environment Information: (Please give the env information, log link or any useful information for this issue)**

### PR DESCRIPTION
**Pull request by anvithks**
**What type of PR is this?**
/kind new feature

**What this PR does / why we need it**:
This PR enables the user to configure the performance metrics collection settings for a storage device.
The user will be able to : 
- Enable performance collection
- Set the scrape interval
- Enable history

**Which issue(s) this PR fixes**:
Fixes #404 

**Test Report Added?**:
 /kind TESTED

**Test Report**:
config-perf-collection-context-menu
config-perf-collection-menu
config-perf-collection-popup
config-perf-collection-success

**Special notes for your reviewer**:

### CODE REVIEW

File: src/app/business/delfin/delfin.service.ts [hunk 1]
```diff
@@ -152,5 +152,10 @@ export class DelfinService {
     deleteAlerts(id, seqNum): Observable<any>{
         return this.http.delete(this.delfinStoragesUrl + '/' + id + '/alerts/' + seqNum);
     }
+
+    //Configure performance metrics collection
+    metricsConfig(id, params): Observable<any>{
```
Comment (NajmudheenCT) [inline src/app/business/delfin/delfin.service.ts:157]: Few comments on the congiguration page
- change scrape interval to polling interval , also mention the value is in seconds 
- Change 'Enable History' to Enable Historic metric collection, ( also if possible provide a tool tip like ' historic collection is the collection all availabe timestamps from previous polling to current polling, also supported only if the backend supports historic collection'

Comment (NajmudheenCT) [reply src/app/business/delfin/delfin.service.ts:157]: Do you have any change for above comment, could not see it in screen shots..

Comment (anvithks) [reply src/app/business/delfin/delfin.service.ts:157]: @NajmudheenCT  These are updated. 
Screenshot below.
image

Comment (anvithks) [reply src/app/business/delfin/delfin.service.ts:157]: @NajmudheenCT I believe the changes have gone in a subsequent PR #481 
I will update this PR to add the changes.

Comment (anvithks) [reply src/app/business/delfin/delfin.service.ts:157]: Updated the PR.

File: src/app/business/delfin/storages/storages.component.ts [hunk 2]
```diff
@@ -93,6 +95,12 @@ export class StoragesComponent implements OnInit {
 
     };
 
+    perfMetricsConfigFormLabel = {
+        "perf_collection": "Enable Performance collection?",
```
Comment (kumarashit) [inline src/app/business/delfin/storages/storages.component.ts:99]: Can you please check and confirm if putting a '?' is fine from Ux perspective? As this is Bool (Yes/No), it's quite obvious

Comment (anvithks) [reply src/app/business/delfin/storages/storages.component.ts:99]: Wil fix this in the subsequent PRs as this will be updated in the next PR

**Review by kumarashit — APPROVED** [review]
LGTM

**Review by NajmudheenCT — APPROVED** [review]
LGTM

**Comment by anvithks** [comment]
@skdwriting Please review and merge

**Review by skdwriting — APPROVED** [review]
LGTM

**Pull request merged**